redbase/implementation/redbase-pf.md

```
@@ -0,0 +1,184 @@
+# The Paged File Component RedBase Part 0: PF
+
+Source: <https://web.stanford.edu/class/cs346/2015/redbase-pf.html>
+
+- [Introduction](#intro)
```

markdownlint-cli2 (0.22.0)

[warning] Link fragments should be valid

(MD051, link-fragments)